.config/typos.toml

```diff
@@ -0,0 +1,60 @@
+# See https://github.com/crate-ci/typos/blob/master/docs/reference.md to configure typos
+
+[files]
+extend-exclude = [
+    "deps/",
+    "src/crc16_slottable.h",
+    "src/eval.c",
+]
+
+[default.extend-words]
+optin = "optin"
+exat = "exat"
+smove = "smove"
+
+[type.c]
+extend-ignore-re = [
+    "BA3E2571", # sha1.c
+    "D4C4DAA4", # sha1.c
+    "Georg Nees"
+]
+
+[type.tcl]
+extend-ignore-re = [
+    "DUMPed"
+]
+
+[type.sv.extend-identifiers]
+# sv = .h
+module_gil_acquring = "module_gil_acquring"
+
+[type.c.extend-identifiers]
+clen = "clen"
+fle = "fle"
+nd = "nd"
+ot = "ot"
+module_gil_acquring = "module_gil_acquring"
```

Comment:
this is in an `extern` variable type, so I didnt want to rename it without discussion.

However, especially given valkey is still unstable, maybe this is a good time to do the rename.

Does valkey want to retain ABI compatibility?

Reply:
I see https://github.com/valkey-io/valkey/issues/43

Reply:
Note this isnt listed in the codespell wordlist.